theme/islandora-dublin-core-description.tpl.php

```diff
@@ -15,6 +15,6 @@
 <div class="islandora-metadata-sidebar">
   <?php if (!empty($dc_array['dc:description']['value'])): ?>
     <h2><?php print $dc_array['dc:description']['label']; ?></h2>
-    <p property="description"><?php print $dc_array['dc:description']['value']; ?></p>
+    <p property="description"><?php print nl2br(filter_xss($dc_array['dc:description']['value'])); ?></p>
```

Comment:
I'm thinking this should possibly be pushed up into the [`islandora_preprocess_islandora_dublin_core_description()`](https://github.com/Islandora/islandora/blob/7.x/theme/theme.inc#L634-L649) (really, that function should probably be named `template_preprocess_islandora_dublin_core_description()`, but that's another issue) or an implementation of either `template_` or `hook_process_islandora_dublin_core_description()`.

... Doing the filter here, after the `if` above... Could potentially end up generating an empty `<p property=description/>` tag? Probably unlikely, but assuming things would get encoded and decoded correctly, a description of `<script of sorts>`... could potentially end up being filtered to the empty string?... Something to test, in any case.